src/ess/powder/grouping.py

```diff
@@ -12,23 +14,42 @@
 )
 
 
+def _drop_grouping_and_bin(
+    data: sc.DataArray, *, dims_to_reduce: tuple[str, ...] | None = None, edges: dict
+) -> sc.DataArray:
+    all_pixels = data if dims_to_reduce == () else data.bins.concat(dims_to_reduce)
```

Comment:
```suggestion
def _drop_grouping_and_bin(
    data: sc.DataArray, *, edges: dict
) -> sc.DataArray:
    all_pixels = data.bins.concat(dims_to_reduce)
```
because `dims_to_reduce` is never used.

Reply:
I was using it for the two-theta case, but then refactored. Since I wanted to consider moving this to ESSreduce I thought I'd keep it.